topics/racket/index.md

```diff
@@ -1,14 +1,15 @@
 ---
-aliases: rkt
+aliases: rkt, racket-lang, racket-language, racketscript, urlang
 created_by: PLT Inc.
 display_name: Racket
 github_url: https://github.com/racket
 logo: racket.png
-related: drracket, language
+related: drracket, language, typed-racket
 released: 1995
 short_description: Racket is a general-purpose programming language and system for language-oriented programming.
 topic: racket
 url: https://racket-lang.org/
 wikipedia_url: https://en.wikipedia.org/wiki/Racket_(programming_language)
 ---
-Racket is a general-purpose, multi-paradigm programming language. It is designed to be a platform for programming language design and implementation.
+Racket is a general-purpose, programming language and a platform for programming language design and implementation. 
+Racket supports multiple paradigms; functional programming, with static type checking, type classes and gradual typing, lazy evaluation; object-oriented programming with classes, mixins, and traits; Functional Reactive Programming (FRP) and language Oriented Programming with powerful macros and parsing libraries.
```

Comment:
Can this be split into multiple sentences? There's too many `;` and `,` here. Also, should language Oriented Programming be capitalised differently?

Reply:
U

Reply:
U

Reply:
U

Reply:
U